pkg/crypto/age.go

```diff
@@ -117,9 +117,20 @@ func SetupAge(client api.Client, emails []string) error {
 			return err
 		}
 
+		dedupeKey := func(id *AgeIdentity) string { return fmt.Sprintf("%s::%s", id.Email, id.Key) }
+
 		idents := ageConfig.Identities
+		present := map[string]bool{}
```

Comment:
"k8s.io/apimachinery/pkg/util/sets"

present := sets.sets.NewString

Reply:
```
		present := sets.NewString()
		for _, id := range idents {
			present.Insert(dedupeKey(id))
		}

		for _, key := range keys {
			id := &AgeIdentity{Key: key.Content, Email: key.User.Email}
			if !present.Has(dedupeKey(id)) {
				idents = append(idents, id)
			}
			
		}

```

Reply:
oh cool, i was hoping someone would build this